user-manual/access-content/search-atom.rst

```diff
@@ -686,31 +687,31 @@ entity. Basic instructions for each are included below.
 .. |searchreset| image:: images/search-resetbutton.png
    :height: 19
 
-A dedicated search box for :term:`authority records <authority record>` has
-been provided on the authority record :ref:`browse page <page-type-browse>`.
-In AtoM 2.0.0, this search box will search the following fields of an
-authority record:
+When browsing :term:`authority records <authority record>` users have two
+options: the dedicated search box and the advanced search options screen. All
+indexed fields can be searched in the dedicated search box including:
 
-* **Authorized form of name** in the Identity area
+* **All forms of name** in the Identity area as well as
 * **All fields** in the Description area (Dates of existence, History, Places,
   Legal status, Functions occupations and activities, Mandates/sources of
   authority, General context)
+* **Access points**: occupation and occupation notes, subjects, places
 * **Dates of creation, revision, and deletion** and **Sources** in the Control
   area
 
-.. NOTE::
+Further search options  to filter results using the advanced search options:
```

Comment:
double-check the spacing of the elements under the NOTE here? I think they need to be indented 3 spaces, to line up with the N of NOTE. I also think we usually add a line break between the admonition header and the note content.

Reply:
That line, NOTE, was deleted.

Reply:
Haha yr right; sorry!